packages/safety-suite-api/src/api/dashboardSink/types.ts

```diff
@@ -42,11 +42,18 @@ export interface KpiTime {
   frame?: KpiTimeFrame;
   interval?: KpiTimeInterval;
 }
+
+export interface KpiNormalization {
+  metric: string;
```

Comment:
```json
{
  "metric": required string,
  "aggregation": required enum,
  "timeField": optional string,
  "filter": optional string
}
```
basically, we want that structure

Reply:
https://idelic.atlassian.net/browse/SAF-6157 mentioned  2.a of ACs

Reply:
I added the optional timeField.